### ISSUE TITLE
"Using the radio role": page is incomplete

### ISSUE THREAD

**Issue by StomperHK** [mdn/content#10066]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_radio_role

#### What information was incorrect, unhelpful, or incomplete?
The site appears to be imcomplete, missing a lot of useful informations.

#### Specific section or headline?
No.

#### What did you expect to see?
A more detailed explanation about the radio role.

#### Did you test this? If so, how?
No.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/aria_techniques/using_the_radio_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_radio_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html
* Last commit: https://github.com/mdn/content/commit/bae04313dc3e19e4a5df529b97d5548c91f39b1f
* Document last modified: 2020-12-30T11:54:07.000Z

</details>

**Comment by estelle** [comment]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role
